### PR TITLE
Export `admin` and `users` groups as `databricks_group` data, not resource

### PR DESCRIPTION
**Pull request by rohitnijhawan**
Fix for issue #1654

TF exporter creatingdatabricks_group resource, not data block for admins and users

### CODE REVIEW

**Review by alexott — APPROVED** [review]
lgtm, although we need to fix the broken test

**Comment by alexott** [comment]
Replace `r` in line 118 of `importables_test.go` with:

```
	d := scim.ResourceGroup().TestResourceData()
	d.Set("display_name", "foo")
	r := &resource{
		Value:     "foo",
		Attribute: "display_name",
		Data:      d,
	}
```

**Review by nfx — CHANGES_REQUESTED** [review]
looks good. please make sure that build is passing

File: exporter/importables.go [hunk 1]
```diff
@@ -560,18 +560,24 @@ var resourcesMap map[string]importable = map[string]importable{
...
-				r.Data.State().Set(&terraform.InstanceState{
-					ID: r.ID,
-					Attributes: map[string]string{
-						"display_name": r.Name,
-					},
-				})
-			} else if r.Data != nil {
-				r.Data.Set("force", true)
+			if r.Data != nil {
+				groupName := r.Data.Get("display_name").(string)
```
Comment (nfx) [inline exporter/importables.go:564]: could you please add unit tests for this change, so that the following commits won't break it?

**Review by nfx — CHANGES_REQUESTED** [review]

File: exporter/importables.go (hunk 1 above)
Comment (rohitnijhawan) [reply exporter/importables.go:564]: I can do that @nfx 

What should the unit test be looking for? An r.Data that **is nil**?
Before we decided to fix this bug, that was the original if condition.

Thanks

**Comment by alexott** [comment]
Really, the correct fix was not adding `if r.Data != nil`, but just fixing a test as I've shown here: https://github.com/databricks/terraform-provider-databricks/pull/1721#issuecomment-1301742079

**Review by nfx — APPROVED** [review]

**Pull request merged**